backend/metricController.js

```diff
@@ -1,5 +1,11 @@
 'use strict';
 
+/**
+ * metricController
+ * @module
+ * @param {JSON} dbConf Database configuration
```

Comment:
There is no JSON type in javascript. 
There's an object type. 

Please also add object contents 

for example: 

@param {object} dbConf Database configuration 
@param {string} dbConf.url url to db.